docs/monitoring.md

```diff
@@ -248,6 +248,27 @@ Security options for the Spark History Server are covered more detail in the
         written to disk will be re-used in the event of a history server restart.
     </td>
   </tr>
+  <tr>
+    <td>spark.history.custom.executor.log.url</td>
+    <td>(none)</td>
+    <td>
+        Specifies custom spark executor log url for supporting external log service instead of using resource managers' app log urls.
+        Spark will support some path variables via patterns which can be vary on resource managers. 
+        Please check the document for resource manager to refer which patterns are supported.
+        <p/>
+        For now, YARN mode only supports this configuration.
+    </td>
+  </tr>
+  <tr>
+    <td>spark.history.custom.executor.log.url.applyIncompleteApplication</td>
+    <td>false</td>
+    <td>
+        Specifies whether apply custom spark executor log url to incomplete applications as well.
```

Comment:
whether to apply  (missing "to")

also as above, add "Even if this is true, this still only affects the behavior of the history server, not running spark applications."